src/lib.rs

```diff
@@ -68,7 +68,13 @@ pub fn sunrise_sunset(
     );
 
     (
-        solar_day.event_time(SolarEvent::Sunrise).timestamp(),
-        solar_day.event_time(SolarEvent::Sunset).timestamp(),
+        solar_day
```

Comment:
Not a maintainer however, the deprecation message for this function describes `SolarEvent` so:
> note = "Use `SolarEvent` which is infaillibe, more flexible and explicit."

With this change I think it might good to remove "infallibe,"

Reply:
Instead of removing it, what would you think of "panic-free"?

Reply:
Perfect.